tests/saitests/py3/sai_qos_tests.py

```diff
@@ -2733,7 +2739,9 @@ def runTest(self):
             pkts_num_egr_mem = int(self.test_params['pkts_num_egr_mem'])
 
         # Pause egress of dut xmit port
-        self.sai_thrift_port_tx_disable(self.dst_client, self.asic_type, [self.dst_port_id])
+        # Disable all dst ports
```

Comment:
Also, Can we have this specific to dnx chassis ?

Reply:
done